### ISSUE TITLE
Ingame status

### ISSUE THREAD

**Issue by Trice1984** [DNA-styx/HLStatsX-Windmill-Theme#22]
Hello,  when i type ingame     status  then load the ingame browser not the correct display

![status](https://github.com/user-attachments/assets/21662ab3-c871-4bd5-ba85-b1e00def9ace)

**Comment by DNA-styx** [comment]
Thanks for reporting.

I'm working through the ingame pages now.

![image](https://github.com/user-attachments/assets/38eee107-60ae-440b-bb38-9ac8b16a9708)

**Comment by DNA-styx** [comment]
That's the first sweep through them. Still have menus to theme and fix a couple of HLStatsX bugs I've found along the way :)

![image](https://github.com/user-attachments/assets/14cba1da-bf38-4f10-8e09-d311acbfce97)
![image](https://github.com/user-attachments/assets/69d7763a-b01e-43a3-9dca-4412dd91e6a0)
![image](https://github.com/user-attachments/assets/82f66614-04f1-4263-9255-d134a05a1a44)

**Comment by DNA-styx** [comment]
That should be all the in-game pages done. Can you give them a test?

**Comment by Trice1984** [comment]
What ingame commands give it?

**Comment by DNA-styx** [comment]
Any of these:

![image](https://github.com/user-attachments/assets/71682bd1-9548-43be-b706-2d4d7044a4a0)

**Comment by DNA-styx** [comment]
I'll close this off. Any bugs/missing features let me know in a new issue

**Comment by Trice1984** [comment]
Yes sorry, its ok. but Teamspeak Server not display, i search the fail at my root.
